v2/operator_claims.go

```diff
@@ -94,9 +94,6 @@ func (o *Operator) Validate(vr *ValidationResults) {
 			vr.AddError("%s is not an account public key", o.SystemAccount)
 		}
 	}
-	if _, _, _, err := ParseServerVersion(o.AssertServerVersion); err != nil {
```

Comment:
We should probably remove ParseServerVersion() then?

Reply:
Removed